<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>quizSection</class>
 <widget class="QWidget" name="quizSection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QuizSection</string>
  </property>
  <widget class="QStackedWidget" name="Pages">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>823</width>
     <height>564</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="MainMenu">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lblInfo">
       <property name="text">
        <string>Welcome to Quiz! There are two options in this section:

1-Level Mode: 

*There are 5 different levels in this mode.

*As you level up, the questions in the levels become more difficult. You must get all the questions right to pass a level.

2-Challenge Mode: 

*In this mode there are 50 questions and 5 minutes to solve them.

*There is also a score keeper and a high score section.

*You will be penalized -1 point for every second that passes and -3 points for every mistake you make.

*You will get +5 points for every question you get right. You can give up if you wish.

*But if you give up, every question you fail will be recorded as incorrect. </string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1111111</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="MakeChoice">
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="leftMargin">
      <number>100</number>
     </property>
     <property name="topMargin">
      <number>100</number>
     </property>
     <property name="rightMargin">
      <number>100</number>
     </property>
     <property name="bottomMargin">
      <number>100</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,1,0">
       <item>
        <widget class="QLabel" name="lbl">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>128</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Roboto Slab</family>
           <pointsize>20</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Please make a choice:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnLevelMode">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>128</horstretch>
           <verstretch>48</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>128</width>
           <height>48</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>11111111</width>
           <height>68</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(82, 148, 226);</string>
         </property>
         <property name="text">
          <string>Level Mode</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnChallangeMode">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>128</horstretch>
           <verstretch>48</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>128</width>
           <height>48</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>11111111</width>
           <height>68</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(82, 148, 226);</string>
         </property>
         <property name="text">
          <string>Challange Mode</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnBack">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1111111</width>
           <height>32</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="ChallangeChoice">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <property name="leftMargin">
      <number>160</number>
     </property>
     <property name="topMargin">
      <number>120</number>
     </property>
     <property name="rightMargin">
      <number>160</number>
     </property>
     <property name="bottomMargin">
      <number>120</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="btnChoice">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>48</height>
          </size>
         </property>
         <property name="text">
          <string>Start Challange!</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnChoice2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>48</height>
          </size>
         </property>
         <property name="text">
          <string>Show Highest Score</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnBack_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1111111</width>
           <height>32</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="LevelChoice">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="leftMargin">
      <number>120</number>
     </property>
     <property name="topMargin">
      <number>120</number>
     </property>
     <property name="rightMargin">
      <number>120</number>
     </property>
     <property name="bottomMargin">
      <number>120</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btnLevel">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>48</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Roboto Slab</family>
           <pointsize>16</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Level 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnLevel2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>48</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Roboto Slab</family>
           <pointsize>16</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Level 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnLevel3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>48</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Roboto Slab</family>
           <pointsize>16</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Level 3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnLevel4">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>48</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Roboto Slab</family>
           <pointsize>16</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Level 4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnLevel5">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>48</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Roboto Slab</family>
           <pointsize>16</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Level 5</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
